### PR TITLE
DOC: Add document description to Eventify

### PR DESCRIPTION
**Pull request by ordirules**
quick doc change

### CODE REVIEW

File: shed/event_streams.py [hunk 1]
```diff
@@ -1282,6 +1282,9 @@ class Eventify(EventStream):
         If none supplied, then load all start keys into event
     output_info: list of tuples, optional
         describes the resulting stream
+    document : {'start', 'descriptor', 'stop}, optional
+        The document to generate events from. Defaults to 'start'
```
Comment (CJ-Wright) [inline shed/event_streams.py]: :+1:

**Comment by CJ-Wright** [comment]
just need a rebase

**Comment by ordirules** [comment]
rebased

**Comment by ordirules** [comment]
thanks!

**Pull request merged**